nimbus/src/nimbus.idl

```diff
@@ -34,7 +34,7 @@ enum Error {
     "RequestError", "ResponseError", "InvalidExperimentResponse"
 };
 
-interface Experiments {
+interface Nimbus {
```

Comment:
I don't want to bikeshed this too much, but in general I'm not a fan of cute project codenames leaking into APIs, because they don't help you understand the point of the code unless you're already familiar with it. I think the ship has already sailed on this one though :-)

Would something like `NimbusClient` or similar be more helpful here with building the mental model; knowing that "this is a client for "Nimbus", whatever that is" seems marginally more useful than knowing that "this is a Nimbus, whatever that is".

I won't object either way though.

Reply:
I really have no objections to naming this NimbusClient (I was also thinking something like NimbusSDK). I'm all for descriptive naming and that would save having to define alias in the A-C component if we also wanted to call it by its "cute" name ;).  I'll go ahead and change this to NimbusClient in this PR since you had a similar issue with this.